googletest/include/gtest/gtest-param-test.h

```diff
@@ -1411,26 +1410,24 @@ internal::CartesianProductHolder10<Generator1, Generator2, Generator3,
 // alphanumeric characters or underscore. Because PrintToString adds quotes
 // to std::string and C strings, it won't work for these types.
 
-#define INSTANTIATE_TEST_CASE_P(prefix, test_case_name, generator, ...)        \
-  static ::testing::internal::ParamGenerator<test_case_name::ParamType>        \
-      gtest_##prefix##test_case_name##_EvalGenerator_() {                      \
-    return generator;                                                          \
-  }                                                                            \
-  static ::std::string gtest_##prefix##test_case_name##_EvalGenerateName_(     \
-      const ::testing::TestParamInfo<test_case_name::ParamType>& info) {       \
-    return ::testing::internal::GetParamNameGen<test_case_name::ParamType>(    \
-        __VA_ARGS__)(info);                                                    \
-  }                                                                            \
+# define INSTANTIATE_TEST_CASE_P(prefix, test_case_name, generator, ...) \
```

Comment:
@dsacre  I don't see any actual changes here, just formatting. If this is the case, could you please revert the formatting changes

Reply:
@gennadiycivil As I tried to explain in the commit message (somewhat briefly), this is not a deliberate formatting change on my part, but rather a fix for a mistake in an older commit. That commit caused the generated header file (`gtest-param-test.h`) to be out of sync with the underlying source file (`gtest-param-test.h.pump`). In other words, I didn't really edit `gtest-param-test.h` here at all, but only re-ran the `pump` script and committed the result.

That said, I was mistakenly under the impression that this was needed as a basis for subsequent changes to these files, which is not the case. So this commit could easily be removed from this pull request. On the other hand, it seems pretty straightforward, and the next person to edit `gtest-param-test.h.*` would have to deal with it anyway.

Reply:
I see. Thank you for the explanation.
Would it be possible to edit the .pump file and re-run the pump to make sure that
1) we only have actual changes , no 'just formatting' changes and 
2) pump and generated .h files are in sync
Thank  you

Reply:
Actually, it is OK,